R/restore.R

```diff
@@ -1,35 +1,19 @@
 
 `_renv_restore_state` <- NULL
 
-#' Restore a Project
+#' Restore project library from the lockfile
 #'
 #' Restore a project's dependencies from a lockfile, as previously generated by
-#' [snapshot()].
-#'
-#' @section Package Repositories:
-#'
-#' By default, the package repositories encoded in the lockfile will be used
-#' during restore, as opposed to the repositories that might already
-#' be set in the current session (through `getOption("repos")`). If you'd like
-#' to override the repositories used by `renv` during restore, you can use,
-#' for example:
-#'
-#' ```
-#' renv::restore(repos = c(CRAN = <...>))
-#' ```
-#'
-#' See also the `repos.override` option in [config] for an alternate way to
-#' provide a repository override.
+#' [snapshot()]. `renv::restore()` compares packages from the lockfile to the
+#' project library, installing any changed packages into the project library.
```

Comment:
Strictly speaking, it compares the packages recorded in the lockfile with all packages available on the library paths, and resolves differences by installing the lockfile-recorded package into the "default" (project) library path.

Not sure if the additional word-smithing is required.

Reply:
I've tried to split the difference so the wording is a bit clearer hopefully without getting too deep into the details.

Reply:
Thanks!